searx/botdetection/limiter.toml

```diff
@@ -0,0 +1,22 @@
+[real_ip]
+
+# Number of values to trust for X-Forwarded-For.
+
+x_for = 1
+
+# The prefix defines the number of leading bits in an address that are compared
+# to determine whether or not an address is part of a (client) network.
+
+ipv4_prefix = 32
+ipv6_prefix = 48
```

Comment:
I have some doubt that this ipv6 prefix is generally correct and that we can find a general correct prefix. My provider for example hands me out /56

Reply:
You can get /48 on some providers so that's why we consider this range to be a single network.

Reply:
> and that we can find a general correct prefix. 

@SuperSandro2000: that is true, but I would prefer /48 prefix over (e.g.) /56 .. there are two use cases we need to consider:

1. A bot in a /48 subnet where a user like you with a /56 prefix is also in (which will probably very rarely be the case).  --> the /48 subnet (and the bot) gets unblocked as soon a /56 user opens SearXNG in the WEB-Browser

2. A bot that owns a complete /48 subnet --> bot is permanently blocked

If we would use a /56 prefix, the bot could run through all its addresses in use case 2. and this would be in my opinion the much worse choice compared to a /48 prefix as described in use case 1.